packages/osd-pm/package.json

```diff
@@ -20,7 +20,6 @@
     "@babel/preset-typescript": "^7.16.5",
     "@node-rs/xxhash": "^1.3.0",
     "@types/cmd-shim": "^2.0.0",
-    "@types/cpy": "^5.1.0",
```

Comment:
did we just sneak this in?

Reply:
no. we `cyp@8.0.0` is already ts. so we should remove `@types/cpy` which also uses `semver`. I think it is a missing update when we bump `cpy`.